scripts/demo-rn-bloat-bundle-test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+
+echo "Bloating the demo project bundle"
+run_f "npm run bloat-bundle"
+
+run_f "npm run build:ios-release"
+run_f "npm run test:ios-release-ci"
+
+echo "Unbloating the demo project bundle"
+run_f "git restore ."
```

Comment:
this could be a problematic way to remove the bloat because the scope is not limited. Perhaps we can write the bloat comments with a unique prefix and then at the end reedit the app.js to remove those lines specifically, or alternatively, make a copy of the app.js and overwrite the bloated bundle at the end with the backup copy?